src/app/pages/login/login-form/login-form.component.scss

```diff
@@ -1,59 +1,73 @@
 .login-form img {
-    width: 1rem;
-    margin: 14px 14px 10px 10px;
-    float: left;
-  }
-  
+  width: 1rem;
+  margin: 14px 14px 10px 10px;
+  float: left;
+}
+
+.login-form {
+  margin: 0rem 0rem 0rem 0rem;
+}
+
+.email, .password {
+  border: none;
+  height: 1.9em;
+  width: 19rem;
+  font-size: 20px;
+}
+
+.email:focus, .password:focus {
+  outline: none;
+}
+
+.input {
+  /* border: 1px solid #f00;; */
```

Comment:
Kindly remove this commented out code too.

Reply:
I have worked on this too